homeassistant/components/repetier/sensor.py

```diff
@@ -35,12 +35,13 @@ def setup_platform(hass, config, add_entities, discovery_info=None):
         printer_id = info["printer_id"]
         sensor_type = info["sensor_type"]
         temp_id = info["temp_id"]
-        name = f"{info['name']}{SENSOR_TYPES[sensor_type][3]}"
+        description = SENSOR_TYPES[sensor_type]
+        name = f"{info['name']}{description.name}"
```

Comment:
```suggestion
        name = f"{info['name']}{description.name or ''}"
```
Allows `desc.name` to be `None` instad of `""`